apk-requirements.txt

```diff
@@ -0,0 +1,2 @@
+libc6-compat
+xz-dev
```

Comment:
We need `xz` instead of `xz-dev`, right? These are supposed to be runtime requirements?

Reply:
I don't really know those details. Let me check...

Reply:
Yep, you are totally right 👍